### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by jyapayne**
Funny story, I was actually working on #181 for a little bit and I was going to mention it on the issue but you got to it before me! So I merged our code and fixed your test in issue181.nim.

### CODE REVIEW

**Comment by dom96** [comment]
Oh nice! So how come my test wasn't working? I skimmed it but I can't identify the magic fix :)

Btw, I would prefer if things like "login" didn't navigate since the login button should be visible from all pages. Same for logout. It might be nice to add a check to make sure that the user isn't already logged in/logged out in those and give a better error message though.

**Comment by jyapayne** [comment]
@dom96 the latest commit depends on https://github.com/dom96/webdriver/pull/4. I will update the nimble file once it has been pulled.

**Comment by jyapayne** [comment]
@dom96, should be good to go unless you have further comments :)

File: tests/browsertests/threads.nim [hunk 1]
```diff
@@ -33,19 +37,57 @@ proc test*(session: Session, baseUrl: string) =
         checkText "#thread-title", userTitleStr
         checkText ".original-post div.post-content", userContentStr
 
-    session.navigate(baseUrl)
-    session.wait()
-    logout(session)
+    session.logout()
+
+proc anonymousTests(session: Session, baseUrl: string) =
```
Comment (dom96) [inline tests/browsertests/threads.nim]: Hrm, this proc depends on ``bannedTests``, wouldn't it make sense to call it in ``bannedTests``, or just merge the two procs?

I'll merge as-is anyway, but something to consider for the next PR :)

**Comment by dom96** [comment]
As always, brilliant work. Thank you!

File: tests/browsertests/threads.nim (hunk 1 above)
Comment (jyapayne) [reply tests/browsertests/threads.nim]: Yep, you're right. I'll do it in the next one. Thanks for the review!

**Pull request merged**